### ISSUE TITLE
human3.6M dataset

### ISSUE THREAD

**Issue by 97chenxa** [kotaro-inoue/human3.6m_downloader#2]
How long have you received their approval? Why have I submitted it for 10 days and asked me to wait patiently?I'm a little hurried to get it.Thank you very much.

**Comment by Ixiaohuihuihui** [comment]
Has your account been approved?

**Comment by 97chenxa** [comment]
No.  I asked them by email. They told me:
Dear prospective user,

Please make sure you supply accurate and certifiable information about you and your research group when you register. Currently, we have no way to make a link between you and your advisor.

Best regards,
The Human3.6M Team

I'm an undergraduate doing a graduate project, not part of a research group, and I wrote the name of our dean, and it still doesn't work, and I don't know what to do.

**Comment by Ixiaohuihuihui** [comment]
Ok, thanks for your reply!

**Comment by 97chenxa** [comment]
Do you have the human3.6M dataset? Can you share with me ? Thank you very much.

At 2019-04-24 17:27:39, "Sycamore" <notifications@github.com> wrote:


Ok, thanks for your reply!

**Comment by Ixiaohuihuihui** [comment]
Nope, I don't have now. I'm sorry for that.

**Comment by ericzw** [comment]
> Do you have the human3.6M dataset? Can you share with me ? Thank you very much. At 2019-04-24 17:27:39, "Sycamore" <notifications@github.com> wrote: Ok, thanks for your reply! — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or mute the thread.

Hi, can you share the dataset to me? Thanks

**Comment by ericzw** [comment]
> No. I asked them by email. They told me:
> Dear prospective user,
> 
> Please make sure you supply accurate and certifiable information about you and your research group when you register. Currently, we have no way to make a link between you and your advisor.
> 
> Best regards,
> The Human3.6M Team
> 
> I'm an undergraduate doing a graduate project, not part of a research group, and I wrote the name of our dean, and it still doesn't work, and I don't know what to do.

Hi, can you share the dataset to me? Thanks

**Comment by lvandoit** [comment]
hi:
  do you have the dataset ,now? can you share it to me?thanks for your any reply. @qiuzhongwei-USTB @97chenxa

**Comment by GalDude33** [comment]
Hi, would appreciate to get the data too.
Thanks advanced! 
@qiuzhongwei-USTB @97chenxa

**Comment by seamanj** [comment]
@97chenxa Have you got the dataset? can you shared with us ? Much appreciated! my email is : tjiang.work@gmail.com

**Comment by 97chenxa** [comment]
I didn't have it. I gave it up. I' m sorry.

At 2019-12-11 05:07:09, "seamanj" <notifications@github.com> wrote:


@97chenxa Have you got the dataset? can you shared with us ? Much appreciated! my email is : tjiang.work@gmail.com

**Comment by seamanj** [comment]
thank you anyway. can I ask u for a favour? can you hide my email in case of receiving some spam mails in the future.  Many thanks for that.

**Comment by alberttjc** [comment]
Hey there, did anyone managed to get their account approved? I tried emailing them but I apparently the email does not exist anymore. Any help will be greatly appreciated!

**Comment by serviceberry3** [comment]
@alberttjc No. I'm an actual researcher working with a professor, but my account hasn't been approved, and I don't see it happening any time soon. I'm not sure what their deal is.

**Comment by VeganEngineer** [comment]
https://drive.google.com/drive/folders/1kgVH-GugrLoc9XyvP6nRoaFpw3TmM5xK

**Comment by philipperbd** [comment]
Hi, dunno if someone will answer but I asked long time ago and still no answer to access dataset. I'm PhD candidate.
